src/site/content/en/blog/optimize-cls/index.md

```diff
@@ -180,138 +259,46 @@ Chrome, Firefox and Safari now support setting `width` and `height` on the `sour
-- Eliminate shifts by reserving the largest possible size for the ad slot.
-  - This works, but it risks having a blank space if a smaller ad creative fills the slot.
-- Choose the most likely size for the ad slot based on historical data.
-
-Some sites may find collapsing the slot initially can reduce layout shifts if the ad slot is unlikely to fill. There isn't an easy way to choose the exact size each time, unless you control the ad serving yourself.
-
-  <figure>
-    {% Video
-      src=["video/tcFciHGuF3MxnTr1y5ue01OGLBn2/bmxqj3kZyplh0ncMAt7x.webm", "video/tcFciHGuF3MxnTr1y5ue01OGLBn2/60c4T7aYOsKtZlaWBndS.mp4"],
-      poster="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/rW77UoJQBHHehihkw2Rd.jpg",
-      controls=true,
-      loop=true,
-      muted=true %}
-   <figcaption>
-      Ads without sufficient space reserved.
-    </figcaption>
-  </figure>
-
-  <figure>
-    {% Video
-      src=["video/tcFciHGuF3MxnTr1y5ue01OGLBn2/tyUFKrue5vI9o5qKjP42.webm", "video/tcFciHGuF3MxnTr1y5ue01OGLBn2/hVxty51kdN1w5BuUvj2O.mp4"],
-      poster="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/rW77UoJQBHHehihkw2Rd.jpg",
-      controls=true,
-      loop=true,
-      muted=true %}
-   <figcaption>
-      Ads with sufficient space reserved.
-    </figcaption>
-  </figure>
+Embeddable widgets allow you to embed portable web content in your page (for example, videos from YouTube, maps from Google Maps, social media posts, and so on). These embeds often aren't aware in advance just how large an embed will be (for example, in the case of a social media post—does it have an embedded image? video? multiple rows of text?). As a result, platforms offering embeds do not always reserve space for their embeds and so cause layout shifts when they finally load.
 
-<figure>
-  {% Img src="image/tcFciHGuF3MxnTr1y5ue01OGLBn2/cX6R4ACb4uVKlUb0cv1c.png", alt="Lighthouse report showing the before/after impact to Cumulative Layout Shift of reserving space for banners like ads", width="800", height="148" %}
-  <figcaption>
-    Lighthouse impact of reserving space for this banner on CLS
-  </figcaption>
-</figure>
+The techniques for dealing with these are all similar. The major differences are how much control you have over the content that will be inserted. If this is inserted by a third-party like an ad partner you may not know the exact size of content that will be inserted, nor be able to control any CLS happening within those embeds.
 
-#### Statically reserve space for the ad slot
+#### Statically reserve space for late-loading content
 
-Statically style slot DOM elements with the same sizes passed to your tag library. This can help ensure the library doesn't introduce layout shifts when it loads. If you don't do this, the library may change the size of the slot element after page layout.
+If placing late-loading content in the content flow, ensure shifts are eliminated by reserving the space for them in the initial layout.
```

Comment:
"Eliminate" is strong language here, as you're not entirely eliminating the layout shift. My vote is for "reduce".

Reply:
Let's go with "can be avoided":

```suggestion
If placing late-loading content in the content flow, layout shifts can be avoided by reserving the space for them in the initial layout.
```